roles/jboss_ws/tasks/main.yml

```diff
@@ -103,7 +81,7 @@
   ignore_errors: yes
   with_items:
     - ."{{ jws_home }}"/httpd/sbin/httpd -v | grep Apache 2>/dev/null
-    - awk -F 'id=' '/JWS/ {print substr($2,16,9)}' "{{ jws_home }}"/tomcat?/webapps/docs/changelog.html 2>/dev/null | grep JWS | head -n1
+    - awk -F 'id=' '/JWS/ {print substr($2,16,9)}' "{{ jws_home }}"/tomcat?/webapps/docs/changelog.html 2>/dev/null | grep JWS | head -n1 
     - cat "{{ jws_home }}"/version.txt  2>/dev/null| grep "Red Hat JBoss Web Server"
     - systemctl status jws5-tomcat.service 2>/dev/null | grep "Apache Tomcat Web" | grep -o jws5 
```

Comment:
Did you decide against pulling out the `systemctl` check to its own task (along with creating one for `chkconfg`?